apps/Hangover/install-64

```diff
@@ -0,0 +1,305 @@
+
+if [ -f "\$WINEPREFIX/system.reg" ];then
+  registry_exists=true
+else
+  registry_exists=false
+fi
+
+export WINEDEBUG=-virtual #hide harmless memory errors
+
+if [ -e "\$WINEPREFIX" ];then
+  status "Checking Wine prefix at \$WINEPREFIX..."
+  echo "To choose another prefix, set the WINEPREFIX variable."
+  echo -n "Waiting 5 seconds... "
+  sleep 5
+  echo
+  # check for existance of incompatible prefix (see server_init_process https://github.com/wine-mirror/wine/blob/884cff821481b4819f9bdba455217bd5a3f97744/dlls/ntdll/unix/server.c#L1544-L1670)
+  # Boot wine and check for errors (make fresh wineprefix)
+  output="\$(set -o pipefail; wine wineboot 2>&1 | tee /dev/stderr; )" #this won't display any dialog boxes that require a button to be clicked
+  if [ "\$?" != 0 ]; then
+    text="Your previously existing Wine prefix failed with an error (see terminal log).
+
+Would you like to remove and regenerate your Wine prefix? Doing so will delete anything you may have installed into your Wine prefix."
+    userinput_func "\$text" "No, keep broken Wine prefix and Exit" "Yes, delete and regenerate Wine prefix"
+    if [ "\$output" == "No, keep broken Wine prefix and Exit" ]; then
+      error "User error: Your current Wine prefix caused Wine to error on launch and you chose to keep it. Manually correct your Wine prefix before installing or updating Wine (x64)."
+    fi
+    warning "Your previously existing Wine prefix failed with an error (see above). You chose to remove it and so it will be re-generated."
+    rm -rf "\$WINEPREFIX"
+    registry_exists=false
+    wine wineboot #this won't display any dialog boxes that require a button to be clicked
+  fi
+  #wait until above process exits
+  sleep 2
+  while [ ! -z "\$(pgrep -i 'wine C:')" ];do
+    sleep 1
+  done
+else
+  status "Generating Wine prefix at \$WINEPREFIX..."
+  echo "To choose another prefix, set the WINEPREFIX variable."
+  echo "Waiting 5 seconds..."
+  sleep 5
+  # Boot wine (make fresh wineprefix)
+  wine wineboot #this won't display any dialog boxes that require a button to be clicked
+  #wait until above process exits
+  sleep 2
+  while [ ! -z "\$(pgrep -i 'wine C:')" ];do
+    sleep 1
+  done
+fi
+
+if [ "\$registry_exists" == false ];then
+status "Making registry changes..."
+TMPFILE="\$(mktemp)" || exit 1
+echo 'REGEDIT4' > \$TMPFILE
+
+#enable font smoothing - see https://askubuntu.com/a/219795
+echo "  - Font smoothing"
+
+MODE=2 # 0 = disabled; 2 = enabled
+TYPE=2 # 1 = regular;  2 = subpixel
+ORIENTATION=1 # 0 = BGR; 1 = RGB
+
+echo '
+[HKEY_CURRENT_USER\Control Panel\Desktop]
+"FontSmoothing"="'\$MODE'"
+"FontSmoothingOrientation"=dword:0000000'\$ORIENTATION'
+"FontSmoothingType"=dword:0000000'\$TYPE'
+"FontSmoothingGamma"=dword:00000578' >> \$TMPFILE
+
+echo "  - Windows 10 appearance theme"
+mkdir -p "\$WINEPREFIX/drive_c/windows/Resources/Themes/Windows_10"
+cp -f /opt/wine-${version}/Windows_10.msstyles "\$WINEPREFIX/drive_c/windows/Resources/Themes/Windows_10"
```

Comment:
see above
Please try to understand out scripts before copy/pasting

Reply:
solved by removing all GUI related registry changes